drivers/netdev_ieee802154/netdev_ieee802154.c

```diff
@@ -63,6 +63,9 @@ void netdev_ieee802154_reset(netdev_ieee802154_t *dev)
 #elif MODULE_GNRC
     dev->proto = GNRC_NETTYPE_UNDEF;
 #endif
+
+    static const uint16_t nid = IEEE802154_DEFAULT_PANID;
```

Comment:
Why not set (and provide in the function below) `dev->pan` here?

Reply:
Changed (reduces flash size by 4 whole bytes on a samr21-xpro  :D )

Reply:
Amazing!!1! :grin: